app/views/mentoring/automation/tooltip_locked.html.haml

```diff
@@ -1,21 +1,20 @@
-.heading.c-generic-tooltip.text-14.w-413
-  %h4.font-semibold.mx-24.mt-20.mb-4 What is automation?
-
-  -# %p.mx-24
-    = ReactComponent::
-    Requires a satisfaction percentage of
-    %strong= @min_satisfaction_percentage.to_s
-    (current: #{@satisfaction_percentage})
-
-  -# %p.mx-24
+.p.heading.c-generic-tooltip.text-14.some{style:'padding:16px 24px;width:413px;'}
+  %p.text-16.text-white.font-semibold.leading-170.mb-12
     Requires
-    %strong= @min_finished_mentoring_sessions.to_s
-    finished mentoring sessions
-    (current: #{@finished_mentoring_sessions})
+    %strong.font-bold= @min_finished_mentoring_sessions.to_s
+    finished mentoring sessions and a satisfaction rating of #{@min_satisfaction_percentage.to_s}% 
+    %br (current rating is #{@satisfaction_percentage}%)
```

Comment:
```suggestion
    %br 
    (current rating is #{@satisfaction_percentage}%)
```

`%br` is a self-closing tag, it doesn't have content.

That said, I don't think you want a `br` here at all really - they're normally a bad idea as they break differently in responsive-world.

Reply:
yep, sorry didn't know the syntax here. 
But since tooltip has a fixed width, I don't think responsiveness has a big effect on this. (or should it?)
Shouldn't it stay?, as it organizes text better, otherwise
<img width="450" alt="Screenshot 2022-09-06 at 10 04 17" src="https://user-images.githubusercontent.com/66035744/188580736-dfdc7439-114d-4d62-a78e-b627af4a5ff0.png">
it would be this 2 half lines, and it looked better to me in a new, clean line.

or should I modify the fixed width of the tooltip?

Reply:
I modified the width of the tooltip, whenever it is under `md` screen-width, it switches to `w-auto`

Reply:
OK, so I think the:
- The text should change to be "95% (currently 0%)"
- The width of the tooltip should change to fit that all onto two lines (so it might need an extra 20px or so wider).